### ISSUE TITLE
Sorting the library - make it more comfortable to use and choose elements

### ISSUE THREAD

**Issue by Netzlichter** [soflyy/oxygen-bugs-and-features#1301]
**Describe the feature you'd like to see included in Oxygen.**

I want: A nice HERO-Design:

At the moment the library is sorted like:
Design Sets ->
- Atomic
- Saas
- Hyperion
- Dentist
- BNB
...

So to choose a nice HERO-Design I have to click through all Design Sets like:
- Atomic -> Sections & Elements -> Heros & Titles
GO BACK and choose another Design Set
- Saas -> Sections & Elements -> Heros & Titles
GO BACK and choose another Design Set
...

It would be great to have: 
Library -> Sections & Elements -> Heros & Titles -> All possible/ predesigned Heros & Titles
Library -> Sections & Elements -> Contact -> All possible/ predesigned Contact
Library -> Sections & Elements -> Content -> All possible/ predesigned Content
.....

It would be much more comfortable to see all predesigned Elements and choose one.

Have a great day :)!

**Comment by nkyb72** [comment]
Please please add this! Would save so much time by element instead of design set. :)

**Comment by michylee** [comment]
I agree. This change would save us a lot of time searching through the various Design Sets.
